cli/mock_app.php

```diff
@@ -49,9 +49,7 @@
     'help' => false,
     'url' => false,
     'token' => false,
-], [
-    'h' => 'help'
-]);
+], ['h' => 'help']);
```

Comment:
personal choice but I'm not a fan of this

Reply:
That's fine. What were your reasons in particular?

The sniff I used in particular was such that arrays that only contained one element, would always be on a single line. This works in the majority of situations quite nicely, and this is the out-of-the-box fixer for such scenarios.

```
Multi-line array contains a single value; use single-line array instead (Squiz.Arrays.ArrayDeclaration.MultiLineNotAllowed)
```

Reply:
The intent behind the code sniff is to make diffs smaller if you add stuff. So if you went and added back in 'u' => 'url' then what would that diff look like? You'd have to redo it all back again to the way it was.

I think this diff should add a trailing , after help and thats it (or bonus points add both url and token shortcuts and then its all moot)